libs/common/src/middleware/api-error-handling.middleware.ts

```diff
@@ -0,0 +1,14 @@
+import { ErrorRequestHandler } from 'express';
+import { ApiError } from '../errors';
+
+/**
+ * Middleware to handle errors of the ApiError type at a global level
+ * Note: Express 4.x requires the route function to be wrapped in a try catch block
+ * and to call `next(error)` to pass the error for this middleware to work
+ */
+export const apiErrorHandling: ErrorRequestHandler = (error, req, res, next) => {
```

Comment:
* Please add types to arguments.
```typescript
import { ErrorRequestHandler, Request, Response, NextFunction } from 'express';

error: unknown, req: Request, res: Response, next: NextFunction
```
* This function doesn't need to explicitly return anything. However, it's good practice to annotate the function correctly for clarity and type safety. i.e. `(error, req, res, next): void`